feat (Animetrain): add presence

Anto-s opened the pull request:
**Describe the changes in this pull request:**

<details> -
<summary>
--
</summary> 
<br>
<!-- Required when a presence has been added or modified ---> -
<!-- Attach screenshot(s) here --->

 ![](https://cdn.discordapp.com/attachments/927653556687867956/945047269332369468/unknown.png)

![](https://cdn.discordapp.com/attachments/927653556687867956/945047444008353842/unknown.png)


</details>

dependabot[bot] committed:
chore(deps-dev): bump @babel/core from 7.17.2 to 7.17.4 (#5815)

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.17.2 to 7.17.4.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.17.4/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Slowlife01 committed:
feat(HBO Max): cover art (#5795)

* feat(HBO Max): cover art

* refactor(HBO Max): rename function

Slowlife01 committed:
fix(Edgenuity): not showing (#5817)

critamine committed:
feat(ArchLinux): add presence (#5762)

* feat(ArchLinux): added basic presence skeleton

* types(Root): replaced nondescript webPath types

* refactor(ArchLinux): added support for subdirs

* refactor(ArchLinux): added support for states

* refactor(ArchLinux): removed unused vars

* feat(ArchLinux): expanded presence support

* Update metadata.json

Corrected version to 1.0.0 for production

* Update websites/A/ArchLinux/presence.ts

Co-authored-by: Bas950 <bas.van.zanten007@gmail.com>

* Update websites/A/ArchLinux/presence.ts

Co-authored-by: Bas950 <bas.van.zanten007@gmail.com>

* refactor(ArchLinux): combined with previous const

* Update websites/A/ArchLinux/presence.ts

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

* fix(ArchLinux): corrected no-unused-vars error

Co-authored-by: Bas950 <bas.van.zanten007@gmail.com>
Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

kyrie25 committed:
feat(Fbox.to): fix presence & add poster image (#5818)

* feat(Fbox.to): fix presence & add poster image

* perf: use type on `.querySelector`s

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

* perf: use types on `.querySelector`s

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

kyrie25 committed:
feat(FMovies): add presence (#5823)

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump @babel/core from 7.17.4 to 7.17.5 (#5826)

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.17.4 to 7.17.5.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.17.5/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

down-bad committed:
fix(YouTube): use regex for image rescaling (#5824)

* fix: use regex for image rescaling

* bump

* increment minor version

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

nayumiDEV committed:
feat(HVNC): add presence (#5801)

* feat(HVNC): add presence

* Update metadata.json

* feat(HVNC): add presence

* feat(HVNC): add presence

paivajz committed:
feat(Booyah): add presence (#5810)

Co-authored-by: Slowlife <slowlife1165@gmail.com>
Co-authored-by: Rodry <38259440+ImRodry@users.noreply.github.com>
Co-authored-by: Rhys <64903135+EncryptedDev@users.noreply.github.com>

nayumiDEV committed:
feat(9GAG): show more presence, fix user page presence, fix top page (#5821)

* show more presence, fix user page presence

* Update presence.ts

* Update metadata.json

* Update presence.ts

* Update presence.ts

itsmelouis committed:
feat(Reverso): add presence (#5825)

Co-authored-by: Nam Anh <phamnamanh25@gmail.com>
Co-authored-by: Nam Anh <77577746+kyrie25@users.noreply.github.com>

Slowlife01 committed:
feat(Notion): page icon (#5827)

Slowlife01 committed:
feat(Autocode): settings and patch (#5828)

Slowlife01 committed:
feat(BBC): small rewrite (#5829)

Faelayis committed:
feat(Hareshi): add presences (#5830)

* feat(Hareshi): add presences

* fix(Hareshi): services failed to validate

* Update websites/H/Hareshi/presence.ts

Co-authored-by: Slowlife <slowlife1165@gmail.com>

Co-authored-by: Slowlife <slowlife1165@gmail.com>

DarkVillager committed:
feat(Videoland): add presence (#5806)

* First & Final Push

* Made Requested Changes

EveryDayRains committed:
fix(Lolz.guru): fix text position (#5797)

* fix(Lolz.guru): fix text position

* fix(Lolz.guru): fix verison

* Update presence.ts

StrikerFRFX committed:
fix(KickAssAnime): Presence is fully functional again (#5822)

* fix(KickAssAnime): no clue what we did

* Update presence.ts

* Update presence.ts

* Update metadata.json

* Update presence.ts

* Update presence.ts

* Update presence.ts

xn-oah commented:
The title of this PR is "Animetrain" but you're using the 9gag website URL, logo, and thumbnail?

EncryptedDev commented:
Rebase to update your branch and discard changes unrelated to the presence you want to modify

Slowlife01 committed:
fix(Anontpp): title not showing (#5833)

Slowlife01 committed:
fix(Peacock): working incorrectly (#5836)

dependabot[bot] committed:
chore(deps-dev): bump eslint-config-prettier from 8.3.0 to 8.4.0 (#5842)

Bumps [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) from 8.3.0 to 8.4.0.
- [Release notes](https://github.com/prettier/eslint-config-prettier/releases)
- [Changelog](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md)
- [Commits](https://github.com/prettier/eslint-config-prettier/compare/v8.3.0...v8.4.0)

---
updated-dependencies:
- dependency-name: eslint-config-prettier
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump terser from 5.10.0 to 5.11.0 (#5848)

Bumps [terser](https://github.com/terser/terser) from 5.10.0 to 5.11.0.
- [Release notes](https://github.com/terser/terser/releases)
- [Changelog](https://github.com/terser/terser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/terser/terser/compare/v5.10.0...v5.11.0)

---
updated-dependencies:
- dependency-name: terser
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump @typescript-eslint/parser from 5.12.0 to 5.12.1 (#5847)

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 5.12.0 to 5.12.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v5.12.1/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps-dev): bump @typescript-eslint/eslint-plugin (#5846)

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 5.12.0 to 5.12.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v5.12.1/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Slowlife01 commented:
You are on SeMiD branch... Create a new branch or use main.
